Formula/c/cdo.rb

```diff
@@ -27,17 +27,24 @@ class Cdo < Formula
   depends_on "proj"
   uses_from_macos "python" => :build
 
+  on_macos do
+    depends_on "llvm" => :build if DevelopmentTools.clang_build_version <= 1403
+  end
```

Comment:
To actually use this we also need

```ruby
ENV.llvm_clang if OS.mac? && DevelopmentTools.clang_build_version <= 1400
```

in the `install` method.

Reply:
Thanks. Not sure how I forgot that 🤦‍♂️